xml/System.IO/Path.xml

```diff
@@ -1716,7 +1716,9 @@ Paths are resolved by calling the <xref:System.IO.Path.GetFullPath%2A> method be
   
 4.  The Windows directory.  
   
-   
+> [!NOTE]
+>  On Unix sytems the enviornment variable is TMPDIR.
```

Comment:
We should probably change the remarks to say what's the sequence on Windows and then on Unix, instead of having this information on a note.

Reply:
Good idea @mairaw. @sclarson can you make the change? You'd have to modify lines 1711 and 1713 to mention the two possible values for the env. variable, depending on the OS.
I'd give you the suggestion but GitHub does not allow me to suggest changes in those lines (they are 2 lines too far from your current change).